kmir/k-src/mir-types.md

```diff
@@ -448,12 +448,13 @@ TODO: add more domain sorts
                     | "Unit"
                     | Bool
                     | "Never"
+                    | "(" MIRValueNeList ")"
                     | "UNIMPLEMENTED"
 
   syntax RValueResult ::= MIRValue
                         | MIRValueNeList
```

Comment:
I am unsure if this should be an `RValueResult`, likely the `MIRValueNEList` is inside some other structure like a tuple or  a slice. The tuple is seen on line 451, and for a slice it would be `| "[" MIRValueNeList "]"`, so perhaps `MIRValue` is the only `RValueResult`